src/sdk/CMakeLists.txt

```diff
@@ -134,7 +134,9 @@ if(SQL_JAVASDK_ENABLE)
             SOURCES sql_router_sdk.i)
     target_include_directories(sql_jsdk PRIVATE ${JNI_INCLUDE_DIRS})
     target_compile_options(sql_jsdk PRIVATE -w)
-    target_link_libraries(sql_jsdk PRIVATE ${SDK_LIBS} boost_filesystem)
+
+    add_dependencies(sql_jsdk hybridse_jsdk_core_static)
+    target_link_libraries(sql_jsdk PRIVATE "-Wl,-force_load $<TARGET_FILE:hybridse_jsdk_core_static>" ${SDK_LIBS} boost_filesystem)
```

Comment:
`-force_load` is only for macos, linux will use `--whole-archive` and `--no-whole-archive`